cotainr/pack.py

```diff
@@ -281,6 +286,35 @@ def _display_miniforge_license_for_acceptance(self, *, installer_path):
                 "No license seems to be displayed by the Miniforge installer."
             )
 
+    @staticmethod
```

Comment:
This should probably be a `@classmethod`, so if a subclass needs to subclass it for whatever reason, it can.

Reply:
Let's leave that question for a future refactor or extension of this class.